src/main/java/com/comcast/zucchini/AbstractZucchiniTest.java

```diff
@@ -31,18 +37,34 @@
  */
 public abstract class AbstractZucchiniTest {
 
-    private static Logger logger = LoggerFactory.getLogger(AbstractZucchiniTest.class);
-    private TestNGZucchiniRunner runner;
-    static HashMap<String, JsonArray> featureSet = new HashMap<String, JsonArray>();
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractZucchiniTest.class);
+    static Map<String, JsonArray> featureSet = new HashMap<String, JsonArray>();
 
     /* Synchronization and global variables.  DO NOT TOUCH! */
+    private static Object lock = new Object();
     private static Boolean hooked = false;
 
+    /* store the list of contexts here */
+    List<TestContext> contexts;
```

Comment:
It's used at package-level access.  For now, it's not a part of the API, and people inheriting from AZT aren't able to access it anyway.  Providing a getter would be an unnecessary extra step.